my_game.toml

```diff
@@ -17,6 +17,7 @@ PLAYER_SPEED_LIMIT = 5  # px/update
 PLAYER_INVINCIBILITY_SECONDS = 5  # secs
 PLAYER_FIRE_RATE = 0.2  # secs
 PLAYER_SHOT_RANGE = 400  # px
+PLAYER_SHOT_FADE = -2 #alpha
```

Comment:
Please make this a positive value. Then update line 221.

Reply:
Done